overrides/partials/toc.html

```diff
@@ -43,10 +43,10 @@
 
   <!-- Table of contents title and list -->
   {% if toc %}
-
-<script src="https://fast.wistia.com/embed/medias/abvwx45h6q.jsonp" async></script><script src="https://fast.wistia.com/assets/external/E-v1.js" async></script><div class="wistia_responsive_padding" style="padding:56.25% 0 0 0;position:relative;"><div class="wistia_responsive_wrapper" style="height:100%;left:0;position:absolute;top:0;width:100%;"><div class="wistia_embed wistia_async_abvwx45h6q seo=false videoFoam=true" style="height:100%;position:relative;width:100%"><div class="wistia_swatch" style="height:100%;left:0;opacity:0;overflow:hidden;position:absolute;top:0;transition:opacity 200ms;width:100%;"><img src="https://fast.wistia.com/embed/medias/abvwx45h6q/swatch" style="filter:blur(5px);height:100%;object-fit:contain;width:100%;" alt="" aria-hidden="true" onload="this.parentNode.style.opacity=1;" /></div></div></div></div>
-
-
+    <!-- Add Banner
```

Comment:
Would be nice to set a TODO here in the following generic format:
TODO: What you want to do

In this case could be:
TODO: When there should be a banner, uncomment this code below